src/index.js

```diff
@@ -0,0 +1,72 @@
+import './index.css';
```

Comment:
I suggest you remove this line, I never see where it is possible to import CSS into a js file.